api/go.mod

```diff
@@ -0,0 +1,76 @@
+module github.com/openshift/hypershift/api
+
+go 1.21
+
+replace (
+	github.com/openshift/hypershift => ../
+	github.com/openshift/hypershift/api => ./
+)
+
+require (
+	github.com/google/go-cmp v0.6.0
+	github.com/google/gofuzz v1.2.0
+	github.com/openshift/api v0.0.0-20240108220607-355cd2586ec4
+	github.com/openshift/hypershift v0.0.0-00010101000000-000000000000
```

Comment:
how's having a different go module for API of any additional value if we are bringing github.com/openshift/hypershift here along with all the other k8s deps?

Reply:
Hey, it was not finished at that point, now it is as it should.